update setShutterOpen() method

fdrgsp opened the pull request:
minor change in the `setShutterOpen()` method:
 - fix annotations
 - use `propertyChanged()` event  instead of `shutterSet` event. 

This is used in https://github.com/tlambert03/napari-micromanager/pull/161.

ianhi commented:
@fdrgsp does autoshutter need the same?

Also, if we are no longer firing the `shutterSet` event then it should be removed from the events object as well.

fdrgsp commented:
> @fdrgsp does autoshutter need the same?

the autoshutter only needs the `autoShutterSet` signal that we have already implemented.

> Also, if we are no longer firing the `shutterSet` event then it should be removed from the events object as well.

True, I just removed it!
